src/pacer.js

```diff
@@ -121,6 +121,7 @@ let PACER = {
 
         default:
           debug(4, `Assuming servlet ${servlet} is not a docket.`);
+          return false;
```

Comment:
If you're going to eliminate the fallthrough, then you also need to add a newline before the next group of `case`s, since they are blocked by comon fallthrough. i.e. on line 125.

Reply:
Adding blank line.